docs/release-notes/rockcraft-1-11.rst

```diff
@@ -0,0 +1,112 @@
+.. _release-1.11:
+
+Rockcraft 1.11 release notes
+============================
+
+2 May 2025
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.11.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+.. craft-application 4.10.0 -> 5.2.0
+   craft-parts 2.7.0 -> 2.9.0
+   Testing command/init
+   plugins: cargo-use, gradle, maven wrapper
```

Comment:
Is this a TODO?

Reply:
yes, to myself while I was drafting this. Thanks for pointing it out, removed!

Reply:
yes, to myself while I was drafting this. Thanks for pointing it out, removed!